benchmarks/operator_benchmark/benchmark_runner.py

```diff
@@ -10,12 +10,6 @@
 
 from caffe2.benchmarks.operator_benchmark import benchmark_core
 
-import caffe2.benchmarks.operator_benchmark.benchmark_caffe2
-import caffe2.benchmarks.operator_benchmark.benchmark_pytorch
-
-import caffe2.benchmarks.operator_benchmark.ops.add
-import caffe2.benchmarks.operator_benchmark.ops.matmul
-
 """Performance microbenchmarks's main binary.
```

Comment:
Thanks for the fix. But, those deleted modules were actually used in the benchmark.